### ISSUE TITLE
Show inhibitory weights as negative number

### ISSUE THREAD

**Issue by apmon** [emer/emergent#63]
It would be nice if in the netview with r.wt and s.wt inhibitory connections would be distinguishable from excitatory weights.

**Comment by rcoreilly** [comment]
also show abs, rel and eff prjn scale factors in netview, as synapse-level vars.

**Comment by StephenJRead** [comment]
I agree.  Its a little disconcerting to not see whether setting a projection to inhibitory worked properly. 
 I also note that I can't find any clear documentation that the way to set an inhibitory connection has changed from old emergent.  In new emergent you have to set a projection type (which you can find in examples, if you know where to look, but doesn't seem to be documented), whereas in old emergent it was a check box on the ConSpec.

**Comment by rcoreilly** [comment]
btw adding projection arrows as an option, and doing this thing, will be at the top of my now-much-shorter-list, when I get a chance to get back to it..  too much other stuff right now for a few weeks at least..
